README.md

```diff
@@ -1,11 +1,24 @@
 # SDK For Verifiable Credentials in Internet Computer
 
-This repository contains a set of libraries to manage verifiable credentials whether you are an issuer or a relying party. 
+This repository contains a set of libraries to manage verifiable credentials whether you are an issuer or a relying party.
 
 ## Libraries
 
 - [Javascript Library](./js-library/README.md). At the moment only a client for relying parties.
 
+## Projects
+
+- [Dummy Relying Party](./dummy-relying-party/README.md). A canister that provides a dummy relying party to request any kind of credentials.
+- [Dummy Issuer](./dummy-issuer/README.md). A canister is a dummy issuer to issue any kind of credentials.
+
+### Test Projects in Mainnet
+
+There is a deployed version in Internet Computer here: [https://l7rua-raaaa-aaaap-ahh6a-cai.icp0.io/](https://l7rua-raaaa-aaaap-ahh6a-cai.icp0.io/).
+
+You can test it against the dummy issuer deployed here: [https://qdiif-2iaaa-aaaap-ahjaq-cai.icp0.io/](https://qdiif-2iaaa-aaaap-ahjaq-cai.icp0.io/).
```

Comment:
I would rather link to the [dashboard](http://dashboard.internetcomputer.org/canister/qdiif-2iaaa-aaaap-ahjaq-cai).
And maybe explicitly state that it does _not_ have a front-end.

Because I just clicked the link and then got the following error message which confused me for a second:

```
Replica Error: reject code CanisterError, message IC0536: Error from Canister qdiif-2iaaa-aaaap-ahjaq-cai: Canister has no query method 'http_request'..
Check that the method being called is exported by the target canister. See documentation: http://internetcomputer.org/docs/current/references/execution-errors#method-not-found, error code Some("IC0536")
```

Reply:
Good idea! Done